tests/scripts/pulpcore/test_upstream_pulp.sh

```diff
@@ -10,13 +10,13 @@ cleanup() {
 }
 trap cleanup EXIT
 
-expect_succ pulp upstream-pulp create --name "cli_test_upstream_pulp" --base-url "$PULP_BASE_URL" --api-root "/pulp/"
+expect_succ pulp upstream-pulp create --name "cli_test_upstream_pulp" --base-url "$PULP_BASE_URL" --api-root "/123/"
```

Comment:
Nice idea.
I was wondering if the server on the upstream side should validate that there are credentials provided. But now that I think of it, you seem to be able to provide them via `.netrc`. So it would be right to not fail. Is this a feature or a bug?